UsbIpServer/Program.cs

```diff
@@ -358,9 +358,10 @@ void DefaultCmdLine(CommandLineApplication cmd)
 ";
                 DefaultCmdLine(metacmd);
 
-                bool CheckWslInstalled()
+                bool CheckWslInstalled(out WslDistributions distros)
                 {
-                    if (!WslDistributions.IsWsl2Installed())
+                    distros = WslDistributions.CreateAsync(CancellationToken.None).Result;
```

Comment:
It looks like `CheckWslInstalled()` is always called from an async context. Can it be async itself instead of doing sync-over-async with `.Result`?